shared/src/bundles/users/types/user-auth-response-dto.type.ts

```diff
@@ -0,0 +1,7 @@
+type UserAuthResponseDto = {
+    id: number;
+    email: string;
+    token: string;
+};
```

Comment:
```suggestion
type AuthReponseDto = {
    user: UserAuthResponseDto,
    token: string;
};

type UserAuthReponseDto = {
  id: number;
  email: string;
}
```
I think it would be better to send it like this. Obviously should be spited to different files.

Reply:
Create to different files for it